manager.spec.yaml

```diff
@@ -1088,6 +1090,14 @@ components:
         type: boolean
       example: false
 
+    advisory:
```

Comment:
@tkasparek will be supported filtering by multiple advisories?

Reply:
@jdobes as far as I know, only by a single advisory